components/derived-from-table.js

```diff
@@ -0,0 +1,72 @@
+// node_modules
+import PropTypes from "prop-types";
+import Link from "next/link";
+// components
+import { DataGridContainer } from "./data-grid";
+import SortableGrid from "./sortable-grid";
+
+const columns = [
+  {
+    id: "@id",
+    title: "Accession",
+    isSortable: false,
+    display: (source) => {
+      const accession = source.source.accession;
+      return <Link href={source.source["@id"]}>{accession}</Link>;
+    },
+  },
+  {
+    id: "file_set",
+    title: "File Set",
+    display: (cell, meta) => {
+      const fileSet = meta.derivedFromFileSets.find(
+        (fileSet) => fileSet["@id"] === cell.source.file_set
+      );
+      <Link href={fileSet["@id"]}>{fileSet?.accession}</Link>;
+      return fileSet ? (
```

Comment:
This is legal, but throughout the code if we don’t display anything if a variable contains nothing, we do:
```js
return (fileSet && (
  <Link>…</Link>
))
```
…or however prettier wants to format that. So that means only execute the part after the `&&` if `fileSet` contains something.

Reply:
Fixed.